RealmSwift/LinkingObjects.swift

```diff
@@ -436,10 +436,3 @@ internal enum LinkingObjectsBridgingMetadata {
         }
     }
 }
-
-// MARK: Unavailable
```

Comment:
Why was this removed? It provides Xcode fix-its for users who are migrating from Realm 2.* to 3 and still using the deprecated APIs.